skopt/acquisition.py

```diff
@@ -150,7 +150,7 @@ def gaussian_pi(X, model, y_opt=0.0, xi=0.01, return_grad=False):
     values[mask] = norm.cdf(scaled)
 
     if return_grad:
-        if not np.any(mask):
+        if not np.all(mask):
```

Comment:
Sorry but I dont get it. Why returning zeros for all samples in X as soon as one of them has `std==0`?